app/views/posts/index.html.rb

```diff
@@ -0,0 +1 @@
+<h1>Post data will load</h1>
```

Comment:
- [ ] Please also add a view file `show.html.erb` file for the post-show action.